sessions/phpunit.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2019 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<phpunit bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite>
```

Comment:
I'm getting the following error when running `phpunit`:

```
  Line 19:
  - Element 'testsuite': The attribute 'name' is required but missing.
```

Reply:
strange, the name attribute is there

Reply:
NM, I must have been looking at the wrong file